src/themes/teams/components/Radio/radioVariables.ts

```diff
@@ -0,0 +1,11 @@
+import { pxToRem } from '../../../../lib'
+
+export default (siteVars: any) => {
+  const vars: any = {}
+
+  vars.fontSize = siteVars.fontSizeMedium
```

Comment:
actually, just for the sake of simplifying things a bit - maybe we could just hardcode some value here for now, and return to that later. However, just expressing my thoughts on that - please, feel free to reason by yourself what would be better; nevertheless, in case we would like to leave this changes here, we'd better to make decision about names for fontWeight

Reply:
I see that the default fontWeight is set to 500 and we need 400 for parity with Teams existing styles.